src/xmidtsend_rbus.c

```diff
@@ -45,6 +45,10 @@ const char * contentTypeList[]={
 "application/binary"
 };
 
+void printSendMsgData(char* status, char* dest, char* transaction_uuid) {
+	ParodusInfo("Status of event: %s, destination: %s ,transation_uuid: %s\n", status?status:"NULL", dest?dest:"NULL", transaction_uuid?transaction_uuid:"NULL");
```

Comment:
I am not sure if the ternary operator works like if(!NULL), better to explicitly add the check (status != NULL) ? status:"NULL"